server/plugin.go

```diff
@@ -527,5 +507,51 @@ func (p *Plugin) trackWithArgs(name, userID string, args map[string]interface{})
 		args = map[string]interface{}{}
 	}
 	args["time"] = model.GetMillis()
-	p.tracker.TrackUserEvent(name, userID, args)
+	_ = p.tracker.TrackUserEvent(name, userID, args)
+}
+
+func (p *Plugin) SendDailyTelemetry() {
+	args := map[string]interface{}{}
+
+	// Jira instances
+	server, cloud := 0, 0
+	instances, err := p.instanceStore.LoadInstances()
+	if err != nil {
+		p.API.LogWarn("Failed to get instances for telemetry", "error", err)
+	}
+	for _, id := range instances.IDs() {
+		switch instances.Get(id).Type {
+		case ServerInstanceType:
+			server++
+		case CloudInstanceType:
+			cloud++
+		}
+	}
+	args["instance_count"] = server + cloud
+	if server > 0 {
+		args["server_instance_count"] = server
+	}
+	if cloud > 0 {
+		args["cloud_instance_count"] = cloud
+	}
```

Comment:
Do we want to instead unconditionally include these keys with count 0, in order to have telemetry payloads have a consistent structure? Not sure what the pros/cons are of _not_ doing that

Reply:
@sri-byte - not sure if this is a question or you or not - but do we have some (strong) preferences wrt to the 0 vs no value in telemetry payloads?